code/game/gamemodes/scoreboard.dm

```diff
@@ -158,7 +158,19 @@ GLOBAL_VAR(scoreboard) // Variable to save the scoreboard string once it's been
 		damaged_job = H.job
 		damaged_key = H.key
 
-// A recursive function to properly determine the wealthiest escapee
+/// A function to determine the cash plus the account balance of the wealthiest escapee
+/datum/scoreboard/proc/get_score_person_worth(mob/living/carbon/human/H)
+	if(!H.mind)
+		return
+	. = get_score_container_worth(H)
+
+	if(!H.mind.initial_account)
+		return
+
+	var/balance = H.mind.initial_account.credit_balance
+	. += balance
+
+// A recursive function to properly determine the cash on the wealthiest escapee
```

Comment:
```suggestion
/// A recursive function to properly determine the cash on the wealthiest escapee
```
3 slashes for autodoc comment, needed when you're defining a new base level proc